src/main/java/frc/robot/utils/ControllerJSONReader.java

```diff
@@ -0,0 +1,274 @@
+        Iterator<JSONObject> iterator = triggersJSON.iterator();
+        while (iterator.hasNext()) {
+            JSONObject trigger = iterator.next();
+            Trigger t;
+
+            int port = ((Long) trigger.get("controller")).intValue();
+
+            // first case is for xbox controllers, second case is for joysticks
+            switch (((String) trigger.get("button"))) {
+                case "a":
+                case "1":
+                    t = controllers.get(port).button(1);
+                    break;
+                case "b":
+                case "2":
+                    t = controllers.get(port).button(2);
+                    break;
+                case "x":
+                case "3":
+                    t = controllers.get(port).button(3);
+                    break;
+                case "y":
+                case "4":
+                    t = controllers.get(port).button(4);
+                    break;
+                case "leftBumper":
+                case "5":
+                    t = controllers.get(port).button(5);
+                    break;
+                case "rightBumper":
+                case "6":
+                    t = controllers.get(port).button(6);
+                    break;
+                case "back":
+                case "7":
+                    t = controllers.get(port).button(7);
+                    break;
+                case "start":
+                case "8":
+                    t = controllers.get(port).button(8);
+                    break;
+                case "leftJoystickPress":
+                case "9":
+                    t = controllers.get(port).button(9);
+                    break;
+                case "rightJoystickPress":
+                case "10":
+                    t = controllers.get(port).button(10);
+                    break;
+                case "pov0":
+                    t = controllers.get(port).pov(0);
+                    break;
+                case "pov45":
+                    t = controllers.get(port).pov(45);
+                    break;
+                case "pov90":
+                    t = controllers.get(port).pov(90);
+                    break;
+                case "pov135":
+                    t = controllers.get(port).pov(135);
+                    break;
+                case "pov180":
+                    t = controllers.get(port).pov(180);
+                    break;
+                case "pov225":
+                    t = controllers.get(port).pov(225);
+                    break;
+                case "pov270":
+                    t = controllers.get(port).pov(270);
+                    break;
+                case "pov315":
+                    t = controllers.get(port).pov(315);
+                    break;
+                case "povCenter":
+                    t = controllers.get(port).pov(-1);
+                    break;
+                default:
+                    t = null;
+                    break;
```

Comment:
Hey this is a bit of a nitpick but can you send a warning through the driver station and set an empty trigger when a non-existent button is configured? I would rather not have to null-check button assignments in `RobotContainer` or risk crashing over a heat-of-the-moment typo.